packages/modules/viewer-react/src/utilities/index.ts

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { ViewerProps } from "../components/BaseViewer";
+import {
+  iTwinViewerInitializerParamList,
+  ItwinViewerInitializerParams,
+} from "../types";
+
+export * from "./MakeCancellable";
+
+/**
+ * Is the item an object?
+ * @param item
+ * @returns
+ */
+export const isObject = (item: any) => {
+  return Object.prototype.toString.call(item) === "[object Object]";
+};
+
+/**
+ * Is the item a function?
+ * @param item
+ * @returns
+ */
+export const isFunction = (item: any) => {
+  return typeof item === "function";
+};
+
+/**
+ * Are a and b equal (deep)?
+ * @param a
+ * @param b
+ * @returns
+ */
+export const isEqual = (a?: any, b?: any) => {
+  if (!a || !b) {
+    return false;
+  }
+  const aKeys = Object.keys(a);
+  const bKeys = Object.keys(b);
+  if (aKeys.length !== bKeys.length) {
+    return false;
+  }
+
+  for (const prop in a) {
+    const objectCompare = isObject(a[prop]) && isObject(b[prop]);
+    const functionCompare = isFunction(a[prop]) && isFunction(b[prop]);
+    if (
+      (objectCompare && !isEqual(a[prop], b[prop])) ||
+      (functionCompare && a[prop].toString() !== b[prop].toString()) ||
+      (!objectCompare && !functionCompare && a[prop] !== b[prop])
+    ) {
+      return false;
+    }
+  }
+  return true;
+};
+
+/**
+ * Pull out iTwin.js initialization options from a set of options
+ * @param options
+ * @returns
+ */
+export const getInitializationOptions = (options?: ViewerProps) => {
```

Comment:
yeah, but lodash is a bit heavy for just a single function. I'm not a huge fan of adding a large utility package unless we have multiple needs

Reply:
aren't there per-function lodash packages just to prevent adding the whole of lodash while still getting the goodness?
https://www.npmjs.com/package/lodash.pick

Reply:
did not know this existed...guess people were tired of excess....thanks.